cms/templates/darklang/preview_lang.html

```diff
@@ -0,0 +1,43 @@
+## Override the default styles_version to the Pattern Library version (version 2)
```

Comment:
Create a shared template for the majority of this file in common (for example, see static_tab.html in LMS which refers to mathjax_include.html in common).